tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 minversion = 2.4
+envlist = py{2.7,3.6}
 
 [testenv]
+changedir = test
```

Comment:
I will check. 

// Doing in-place tests is usually a bad idea since it doesn't test that you've actually packaged things properly (or messed up the imports), since your code will not be imported the same way as the end users will do it. Most often than not, `changedir` to a folder that doesn't contain `__init__.py` isolates the test code from the the source tree and ensures you're testing against site-packages.